### DISCUSSION TITLE
Notch Filter

### DISCUSSION THREAD

**Discussion by stellarpower** [neuromore/studio#222] [Ideas]
Is there a way to implement a notch filter in Studio? I've tried using a high-order, narrow bandstop, but the response doesn't look to be that close to what I see in other tools where these have a dedicated notch for killing one target frequency.

Thanks

**Comment by JuliusCosmoRomeo** [comment]
Hi @stellarpower, thanks for starting the thread. There are basically two ways to use the notch filter to get rid of the power line frequency:
1. As you said, using a high-order narrow bandstop. Be aware that it shouldn't be too narrow because of the filter onset, even if the fitler order is relatively high. 
2. Via the settings: If you go to `Settings-> Devices` there is an option to filter the local power line frequency. 
Let us know if that helps! 
Julius
